packages/communication-react/CHANGELOG.stable.md

```diff
@@ -4,86 +4,41 @@ This log was last generated on Tue, 16 Jan 2024 22:14:17 GMT and should not be m
 - Ensure the CallComposite and CallWithChatComposite side pane's are closed if the call is rejoined ([PR #3941](https://github.com/azure/communication-ui-library/pull/3941) by 2684369+JamesBurnside@users.noreply.github.com)
 - Fixed an issues when mentions list wasn't scrolled when navigated by keyboard ([PR #3958](https://github.com/azure/communication-ui-library/pull/3958) by 98852890+vhuseinova-msft@users.noreply.github.com)
-- Remove beta banners for stablized features ([PR #3952](https://github.com/azure/communication-ui-library/pull/3952) by 96077406+carocao-msft@users.noreply.github.com)
 - Update camera behavior to turn off the camera befroe we go on hold for privacy reasons ([PR #3926](https://github.com/azure/communication-ui-library/pull/3926) by 94866715+dmceachernmsft@users.noreply.github.com)
-- Fix up Chat sample for noImplicitAny ([PR #4000](https://github.com/azure/communication-ui-library/pull/4000) by 3941071+emlynmac@users.noreply.github.com)
-- Correctly apply tablist role and tablist behavior to sidepane tab headings on mobile ([PR #3955](https://github.com/azure/communication-ui-library/pull/3955) by 2684369+JamesBurnside@users.noreply.github.com)
-- Fix up noImplicitAny for Chat Composites ([PR #3987](https://github.com/azure/communication-ui-library/pull/3987) by 3941071+emlynmac@users.noreply.github.com)
-- Fix up chat components noImplcitAny ([PR #3988](https://github.com/azure/communication-ui-library/pull/3988) by 3941071+emlynmac@users.noreply.github.com)
+- Apply tablist role and tablist behavior to sidepane tab headings on mobile ([PR #3955](https://github.com/azure/communication-ui-library/pull/3955) by 2684369+JamesBurnside@users.noreply.github.com)
 - Update remote screen share component to respect scaling ([PR #3956](https://github.com/azure/communication-ui-library/pull/3956) by 94866715+dmceachernmsft@users.noreply.github.com)
-- Enable the ability for user to edit value in text field with keyboard ([PR #3970](https://github.com/azure/communication-ui-library/pull/3970) by 94866715+dmceachernmsft@users.noreply.github.com)
 - Fix for an issue when a new messages button was shown because the status didn't match for the same message ([PR #3930](https://github.com/azure/communication-ui-library/pull/3930) by 98852890+vhuseinova-msft@users.noreply.github.com)
 - Introduce blocking error when giving wrong kind of identifier to createAzureCommunicationCallAdapter ([PR #3972](https://github.com/azure/communication-ui-library/pull/3972) by 94866715+dmceachernmsft@users.noreply.github.com)
-- Fix for image placeholder issue for GA release ([PR #3911](https://github.com/azure/communication-ui-library/pull/3911) by 98852890+vhuseinova-msft@users.noreply.github.com)
-- Resolve resource not found error for inline image placeholder UITest ([PR #3979](https://github.com/azure/communication-ui-library/pull/3979) by 73612854+palatter@users.noreply.github.com)
```

Comment:
https://github.com/azure/communication-ui-library/pull/3979 actually should be in the changelog.

Reply:
How come? UITests should not be part of changelogs.